### PR TITLE
Changing OCP channel from stable to fast to install OCP 4.4.3

### PR DESCRIPTION
**Pull request by Shrivaibavi**
The change have been made because the OCP have GAed 4.4.3 in fast channel and the stable one provides OCP 4.3.18.
Referencing the mail thread here http://post-office.corp.redhat.com/archives/aos-devel/2020-May/msg00071.html

Signed-off-by: Shrivaibavi Raghaventhiran <sraghave@redhat.com>

### CODE REVIEW

File: conf/ocp_version/ocp-4.4-ga-config.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ RUN:
 DEPLOYMENT:
   ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
   installer_version: "4.4-ga"
-  ocp_channel: "stable"
+  ocp_channel: "fast"
```
Comment (RazTamir) [inline conf/ocp_version/ocp-4.4-ga-config.yaml:10]: @Shrivaibavi could you please add more explanation of why this change?

Comment (Shrivaibavi) [reply conf/ocp_version/ocp-4.4-ga-config.yaml:10]: @RazTamir Added, Please check

Comment (petr-balogh) [reply conf/ocp_version/ocp-4.4-ga-config.yaml:10]: @RazTamir it's cause there is no promoted content to stable-4.4 channel. See mail thread:
http://post-office.corp.redhat.com/archives/aos-devel/2020-May/msg00071.html

@Shrivaibavi you can add comment pointing to this mail thread and the note that once there is build promoted to stable that we can revert back to stable.

Comment (Shrivaibavi) [reply conf/ocp_version/ocp-4.4-ga-config.yaml:10]: @petr-balogh Added comment session

**Comment by Shrivaibavi** [comment]
This PR is about OCP deployment and it passed as expected. 
$ oc get clusterversion
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.4.3     True        False         39m     Cluster version is 4.4.3

The above job failure is because of ceph_health_warn 
>               f"Ceph cluster health is not OK. Health: {health}"
            )
E           ocs_ci.ocs.exceptions.CephHealthException: Ceph cluster health is not OK. Health: HEALTH_WARN clock skew detected on mon.b, mon.c

ocs_ci/utility/utils.py:1350: CephHealthException

File: conf/ocp_version/ocp-4.4-ga-config.yaml [hunk 2]
```diff
@@ -7,7 +7,10 @@ RUN:
...
```
Comment (hnallurv) [inline conf/ocp_version/ocp-4.4-ga-config.yaml:10]: how will you come to know that 4.4 moved to Stable? Do you get notification?

Comment (petr-balogh) [reply conf/ocp_version/ocp-4.4-ga-config.yaml:10]: We will need to check manually.  As was mentioned in mail thread one of web app can easily show you now here:
https://openshift-channels.robszumski.com/ 

Or we can use the command I mentioned in mail thread.
